src/annotator-custom-editor.js

```diff
@@ -7,16 +7,19 @@ function annotatorCustomEditor(options){
 
     // ajax get to retrieve data
     getDataPayload: function(){
-      $.get(this.options.endpoint, (data) => {
-        this.configureData(data);
-      })
+      if (typeof this.options.data !== "object") {
```

Comment:
when it is a URL (string). then we'd get the data and pipe it through. i had it built like this initially so i figured we'd leave it in as a switch.